packages/core/src/api/client/index.ts

```diff
@@ -35,9 +34,7 @@ function createQueryFetcher(
 ) {
   return async function (query, variables): Promise<any> {
     const url = getGqlUrl();
-    const token = getAccessToken({
-      request: context,
-    });
+    const token = getAccessToken();
```

Comment:
Something to think about here, should we always apply the token or potentially enable two clients from core: an `authClient` and `client`? Then in `@faust/next` we could also export an `authClient` and `client` that use the same interface, but you can determine which you need to use when forming your queries?

Reply:
I think we can get away with one client since we can leverage `isAuthenticated` from the `useAuth()` hook

Reply:
The only thing I'm thinking here is that we are always applying the access token to the Authorization header, which ends up with extra work on the WP end as well as a potential for authentication errors even when an authenticated request isn't necessary, right?

Reply:
Yeah, that is a fair point. I suppose logically we shouldn't be adding an access token to a request that doesn't need it.

Reply:
I think an `authClient` and `client` would work, I would just want to make sure that doesn't require use to duplicate any logic within the clients.